chidioguejiofor/ojumo-website-fend
# Prof: first section of landing page

#### What does this PR achieve?
- adds components to a Landing page
- restructures app in a more clean way
- adds a NavigationOption atom
- adds a NavigationOptionGroup to represent all navigation
- adds creates the Header that contains the logo and navigation
- adds a devices file that contains devices constants
- adds different font rems to the app
- moves the reset_styles file

#### Any background context?
-  This is the first section of the Landingpage

#### Manual Testing Steps
- Fetch `ft-prof-first-section-of-landing-page` branch
- Checkout `ft-prof-first-section-of-landing-page` branch
- Start app via `npm run dev`
- You should see this first part clean

#### Screenshots:
<img width="1440" alt="Screenshot 2019-12-05 at 3 26 43 PM" src="https://user-images.githubusercontent.com/38565349/70243841-ad09cd00-1773-11ea-892b-f36fef980b19.png">
<img width="1440" alt="Screenshot 2019-12-05 at 3 26 33 PM" src="https://user-images.githubusercontent.com/38565349/70243843-ada26380-1773-11ea-8f51-6ba62d9cac4d.png">

#### Relevant Trello Tickets
- [FIrst section of LandingPage](https://trello.com/c/PByI7dSD/4-user-should-be-able-to-see-the-first-section-of-the-landing-page)
